uui-components/src/pickers/PickerToggler.tsx

```diff
@@ -49,17 +49,22 @@ function PickerTogglerComponent<TItem, TId>(props: PickerTogglerProps<TItem, TId
     }, [props.isOpen]);
 
     const isActivePlaceholder = (): Boolean => {
-        if (props.isReadonly) return  false;
+        if (props.isReadonly) return false;
         else if (props.isOpen && props.searchPosition === 'input') return false;
         else if (props.minCharsToSearch && inFocus) return false;
         else if (props.pickerMode === 'single' && props.selection && props.selection.length > 0) return true;
         else return false;
     };
 
     const blur = (e?: React.FocusEvent<HTMLElement>) => {
-        setInFocus(false);
-        props.onBlur?.(e);
-        inputContainer.current?.blur();
+        const blurTrigger = e.relatedTarget as HTMLElement;
```

Comment:
@vik753, I've turned back some changes from you previous PR: https://github.com/epam/UUI/pull/804

Could you, please, check if nothing was broken by a current fix. Thanks.